WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController+JetpackPrompt.swift

```diff
@@ -0,0 +1,45 @@
+import Foundation
+
+extension NotificationsViewController {
+
+    func promptForJetpackCredentials() {
+        guard !showingJetpackLogin,
+            let blog = blogService.lastUsedBlog() else {
```

Comment:
Also, this seems to run just once when the view loads. Since this is in its own tab, last used blog my change, or you might even remove it from the app.